### PR TITLE
Added Awesome Portfolio Websites to the list of projects 😎

### PR DESCRIPTION
**Pull request by smaranjitghose**
*** SELF EXPLANATORY***

### CODE REVIEW

File: docs/winter_of_code.md [hunk 1]
```diff
@@ -89,6 +89,19 @@ Incog.ly is a peer to peer video conferencing app which keeps your identity anon
...
+### [Awesome Portfolio Websites](https://github.com/smaranjitghose/awesome-portfolio-websites)
+An open-source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world. We take care of the SEO and Design so that others can work on their projects without worrying about a place to showcase.
+
+Tech Stack:
+- HTML, CSS, Bootstrap, JavaScript (The defacto toolkit of any web dev)
+- React
+- Tailwind ( For building neat custom CSS components)
+- SVG Animations (If you wish to take the challenge)
+
+Project Maintainers: [Smaranjit Ghose](https://www.linkedin.com/in/smaranjitghose/), [Anush Bhatia](https://www.linkedin.com/in/anushbhatia/)
```
Comment (sansyrox) [inline docs/winter_of_code.md]: @smaranjitghose , please don't add personal links here.

File: docs/mentors.md [hunk 2]
```diff
@@ -5,6 +5,11 @@
     <th>Name</th>
     <th>Project</th>
   </tr>
+    <tr>
+    <td>Smaranjit Ghose</td>
```
Comment (sansyrox) [inline docs/mentors.md:9]: You can href your name here if you want to add a reference.

**Review by sansyrox — CHANGES_REQUESTED** [review]
Looks good, please see inline comments.

File: docs/mentors.md (hunk 2 above)
Comment (smaranjitghose) [reply docs/mentors.md:9]: Cool

**Comment by smaranjitghose** [comment]
> Looks good, please see inline comments.

Updated

**Review by sansyrox — CHANGES_REQUESTED** [review]
Thanks. LGTM! 🚀

**Pull request merged**